src/client/components/UserPage/UserLinkTable/ToolBar/SearchInput/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { connect, useSelector } from 'react-redux'
+import {
+  InputAdornment,
+  TextField,
+  createStyles,
+  makeStyles,
+} from '@material-ui/core'
+import debounce from 'lodash/debounce'
+
+import userActions from '../../../../../actions/user'
+import useMinifiedActions from '../util/useMinifiedActions'
+
+const mapDispatchToProps = (dispatch) => {
+  const debouncedUpdateSearchText = debounce(
+    () => dispatch(userActions.getUrlsForUser()),
+    500,
+  )
+  return {
+    updateSearchText: (searchText) => {
+      dispatch(userActions.setUrlTableConfig({ searchText, pageNumber: 0 }))
+      debouncedUpdateSearchText()
+    },
+  }
+}
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    searchInput: {
+      display: 'flex',
+      flex: (props) => (props.fillWidth ? 1 : 'unset'),
+      width: (props) => (props.fillWidth ? 'unset' : 445),
+    },
+    input: {
+      flexGrow: '1',
+      height: '100%',
+      minHeight: (props) => props.textFieldHeight,
+      padding: theme.spacing(0),
+      lineHeight: 1.5,
+    },
+  }),
+)
+
+// Height of the text field in the search input.
+const textFieldHeight = 48
```

Comment:
full caps like `TEXT_FIELD_HEIGHT` for constants like this

Reply:
Solved in commit 7843904.